pkg/cmd/clients.go

```diff
@@ -241,3 +241,121 @@ Run the "mobile get clients" command from this tool to get the client ID.`,
 	}
 	return command
 }
+
+// SetClientValueCmd sets value in client
+func (cc *ClientCmd) SetClientValueFromJsonCmd() *cobra.Command {
+	var patch string
+	command := &cobra.Command{
+		Use:   "client <clientID>",
+		Short: "Patches mobileclient with provided json",
+		Long: `set client allows you to patch a mobile client in your namespace.
+			   Run the "mobile get clients" command from this tool to get the client ID.`,
+		Example: `mobile set client <clientID> --patch='{"spec": {"dmzUrl": "www.dmz.com"}}' --namespace=myproject 
+  			      kubectl plugin mobile set client <clientID> --patch='{"spec": {"dmzUrl": "www.dmz.com"}}'
+				  oc plugin mobile set client <clientID> --patch='{"spec": {"dmzUrl": "www.dmz.com"}}'`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var (
+				err error
+				res *v1alpha1.MobileClient
+				ns  string
+			)
+
+			if len(args) != 1 {
+				return cmd.Usage()
+			}
+			clientID := args[0]
+
+			patch, err := cmd.PersistentFlags().GetString("patch")
+			if err != nil {
+				return errors.Wrap(err, "failed to get patch flag")
+			}
+
+			ns, err = currentNamespace(cmd.Flags())
+			if err != nil {
+				return errors.Wrap(err, "failed to get namespace")
+			}
+
+			res, err = cc.mobileClient.MobileV1alpha1().MobileClients(ns).Patch(clientID, types.MergePatchType, []byte(patch))
+			if err != nil {
+				return errors.Wrap(err, "failed to patch mobile client with clientID "+clientID)
```

Comment:
`fmt.Sprintf("failed to set value in mobile client with clientID %s", clientID)`